src/System.Device.Gpio.Tests/GpioControllerTestBase.cs

```diff
@@ -85,7 +85,7 @@ public void ThrowsInvalidOperationExceptionWhenPinIsNotOpened()
         }
 
         [Fact]
-        public void IsPinOpenTest()
+        public void IsPinOpenOnInputTest()
```

Comment:
One last ask, can you also please change line 93 on this file so that we explicitly open the pin using Input mode? Right now it is not setting any mode which is defaulting to Input, but given the change on the test name, it would be good if we explicitly set the mode when opening.

Reply:
especially this is opening LED which looks like it's opening for output